db/employmentDb.sql

```diff
@@ -1,51 +1,64 @@
-
 CREATE TABLE employee (
-
 	employee_id serial primary key,
 	employee_firstName text not null,
 	employee_lastName text not null,
 	employee_phoneNumber text not null,
 	employee_email text not null,
 	employee_address text not null,
+	employee_role text not null,
```

Comment:
If you make this change to an existing database, what will happen? 

you've added a new text column which cannot be null. What will be the value of this column in all the existing rows?